tools/rows_picker_gen.go

```diff
@@ -0,0 +1,145 @@
+// +build ignore
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"time"
+)
+
+func main() {
+	var err error
+	fn := flag.String("o", "", "output file")
+	flag.Parse()
+
+	out := os.Stdout
+	if *fn != "" {
+		out, err = os.Create(*fn)
+		if err != nil {
+			log.Fatalf("could not create file %q, %v", *fn, err)
+		}
+	}
+	defer out.Close()
```

Comment:
The return value of Close() should be checked, if it is not nil the program should exit with an error.
In the unlikely case that Close() failed the content might have not be written to disk.